spock-specs/src/test/groovy/org/spockframework/smoke/ast/AstSpec.groovy

```diff
@@ -0,0 +1,211 @@
+package org.spockframework.smoke.ast
+
+import org.spockframework.EmbeddedSpecification
+import spock.util.Show
+
+import org.codehaus.groovy.control.CompilePhase
+
+class AstSpec extends EmbeddedSpecification {
+  def "astToSourceFeatureBody renders only methods and its annotation by default"() {
+    when:
+    def result = compiler.astToSourceFeatureBody('''
+    given:
+    def nothing = null
+    ''')
+
+    then:
+    result == '''\
+@org.spockframework.runtime.model.FeatureMetadata(name = 'a feature', ordinal = 0, line = 1, blocks = [org.spockframework.runtime.model.BlockKind.SETUP[]], parameterNames = [])
```

Comment:
The `blocks` value does not look right.
If you look at a decompiled class file of our tests, it looks like
```
blocks = {@BlockMetadata(
    kind = BlockKind.SETUP,
    texts = {"an a"}
)}
```

Reply:
yeah I fixed it now

Reply:
Isn't the `org.spockframework.runtime.model.BlockKind.SETUP[]` in the beginning still wrong?
I think I would have expected
```
blocks = [@org.spockframework.runtime.model.BlockMetadata(kind = org.spockframework.runtime.model.BlockKind.SETUP, texts = [])]
```